src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -136,13 +136,15 @@ export class RecommendationHandler {
     }
 
     async getTaskTypeFromEditorFileName(fileName: string): Promise<CodewhispererGettingStartedTask> {
-        if (fileName.startsWith('CodeWhisperer_generate_suggestion')) {
+        const parts = fileName.split('/')
+        const name = parts[parts.length - 1]
```

Comment:
could `name` be undefined? what guarantees that `fileName` has slashes?

Reply:
Updated